packages/fluentui/react-northstar/src/themes/teams-dark/componentVariables.ts

```diff
@@ -16,5 +16,4 @@ export { default as SvgIcon } from './components/Icon/iconVariables';
 export { default as Reaction } from './components/Reaction/reactionVariables';
 export { default as Alert } from './components/Alert/alertVariables';
 export { default as Provider } from './components/Provider/providerVariables';
-export { default as Dropdown } from './components/Dropdown/dropdownVariables';
```

Comment:
Was this intentional?

Reply:
No longer needed now that we have colorScheme in place